### ISSUE TITLE
Judge uses CPU time or total time?

### ISSUE THREAD

**Issue by guptaaditya13** [sankha93/codejudge#38]
I have doubt, while executing, the judge considers the total time spent by the process or the time spend by the process in CPU. ideally it should be the second one.

**Comment by sankha93** [comment]
Yes, this is a bug that needs to be fixed. This should be the CPU time, but right now it only measures the total time spent by the process.

**Comment by guptaaditya13** [comment]
@sankha93  do you have any idea how we can implement it?

**Comment by sankha93** [comment]
Yes, I think some form of the `ulimit` needs to be set to fix the maximum CPU time of the program.

That said, I do agree there needs to be a better sandbox implementation for this. I have some ideas for it, may be I'll get to implementing it soon.
